### DISCUSSION TITLE
no ODC environment found

### DISCUSSION THREAD

**Discussion by qypeng12** [EY-Data-Science-Program/2021-Better-Working-World-Data-Challenge#45] [3.0 VMs and data (Azure VM, local deployment, and Jupyter)]
I am following the instruction on Azure setup and everything worked well until I tried to query the datacube (attached below).
My database and jupyter server are both "Up", and I found error responses in initializing the datacube system when I check the installation log (see attached screenshot).
Could you kindly inform me of possible ways to get it work? Thanks a lot!

![image](https://user-images.githubusercontent.com/29447145/115324172-46097000-a13e-11eb-8ea5-eee903492645.png)
![image](https://user-images.githubusercontent.com/29447145/115324214-5883a980-a13e-11eb-88f1-0dce02ddc030.png)
![image](https://user-images.githubusercontent.com/29447145/115324242-69ccb600-a13e-11eb-8525-8ab916af1970.png)

**Comment by alexgleith** [comment]
Hey, in Jupyter, can you please show me the output of this command: `!env | grep DB`

You should see a number of database connection details.

**Reply by qypeng12** [reply]
I got zero output using the command :(

**Reply by alexgleith** [reply]
Very strange.

Did you edit the docker-compose.yaml?

What you can do is delete the resource group and creating a new one.

**Reply by qypeng12** [reply]
I didn't edit that - I'll try making a new one. Thanks!
